tests/autologging/fixtures.py

```diff
@@ -29,6 +29,9 @@ def recursive_fn(self, level, max_depth):
             else:
                 return self.recursive_fn(level + 1, max_depth)
 
+        def bad_fn(self, error_to_raise):
```

Comment:
```suggestion
        def throw_fn(self, error_to_raise):
```

Can we rename this function to clarify what it does?